src/qt/pivx/dashboardwidget.cpp

```diff
@@ -301,7 +338,7 @@ void DashboardWidget::onSortTypeChanged(const QString& value)
 
     // Store settings
     QSettings settings;
-    settings.setValue("transactionType", filterByType);
+    settings.setValue("transactionType", filterIndex);
```

Comment:
We should store `filterByType` and not the index. There are types that are not used and the ordering in the combobox could easily change. The `TransactionFilterProxy::TYPE(type)` will not.

Reply:
good idea, i've reworked this a bit to address this